src/components/ContentMapping/ContentMapping.ts

```diff
@@ -37,5 +40,10 @@ export const ContentMapping: ContentMappingType = {
     EXAMPLE_IMAGE: {
         widget: ExampleImage,
         editor: ExampleImageEditor
-    }
-}
\ No newline at end of file
+    },
+    SEPARATOR: {
+    widget: Separator,
```

Comment:
Nitpick: Tab this out

Reply:
Please set your tab width to 4 spaces for now for the entire repo (can be found in vscode settings). I will update this to regular tabs in the future so you can define how much spaces you want your tabs.